pyproject.toml

```
@@ -0,0 +1,93 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "PyPDF2"
+authors = [{ name = "Mathieu Fenniak", email = "biziqe@mathieu.fenniak.net" }]
+maintainers = [{ name = "Martin Thoma", email = "info@martin-thoma.de" }]
+description = "A pure-python PDF library capable of splitting, merging, cropping, and transforming PDF files"
+readme = "README.md"
+dynamic = ["version"]
+license = { file = "LICENSE" }
```

```suggestion
license = BSD-3-Clause
```

See https://python-poetry.org/docs/pyproject/#license